modules/cluster/src/main/java/io/fluo/cluster/runner/ScanOptions.java

```diff
@@ -40,6 +40,10 @@
   @Parameter(names = {"-h", "-help", "--help"}, help = true, description = "Prints help")
   public boolean help;
 
+  @Parameter(names = {"-esc", "--escape-non-ascii"}, help = true,
+      description = "Hex encode non ascii bytes")
+  public boolean hexEncNonAscii = false;
```

Comment:
Should scan escape non-ascii by default?  It is nicer to view but I guess it could confuse user who don't know it is the default.  I wanted to bring this up but I could go either way.